QuasarHelios2015/src/org/team1540/quasarhelios/AutoEjector.java

```diff
@@ -35,9 +35,10 @@ public static BooleanStatus create() {
     protected void autonomousMain() throws AutonomousModeOverException, InterruptedException {
         try {
             setClampHeight(1.0f);
-            Elevator.setBottom.event();
-
-            waitUntil(Elevator.atBottom);
+            if (!Elevator.atBottom.get()) {
+                Elevator.setBottom.event();
+                waitUntil(Elevator.atBottom);
+            }
```

Comment:
Yes, as it keeps waiting for the limit switch to flip even though the elevator is already at the bottom.